test/verify/netlib.py

```diff
@@ -91,7 +91,7 @@ def wait_for_iface(self, iface, active=True, state=None, prefix="10.111."):
             self.browser.wait_in_text(sel, text)
         except:
             print("Interface %s didn't show up." % iface)
-            print(self.browser.eval_js("$('#networking-interfaces').html()"))
+            print(self.browser.eval_js("ph_find('#networking-interfaces').outerHTML"))
```

Comment:
Don't you need to call the function here?

Reply:
IMHO it's preferrable to calling `document.querySelector()`, as this would need to be modified when moving to chromium (as `document` is not the current frame there, needs to move to `this.cur_document`). `ph_find()` is our test suite API (which will be available regardless of which browser backs it), so whenever possible I'd like our tests to call the `ph_*()` functions.

Note that despite the name, test/common/phantom-lib.js and `ph_*()`  are *not* specific to Phantom. In some earlier PR I already renamed the file to test-functions.js (but you wanted to postpone this), and some day we should also rename the `ph_*()` functions to maybe `test_*()` or `ct_*()` ("cockpit test"). Or we just reinterpret it as "page helper" ☺